code/datums/uplink/implants.dm

```diff
@@ -46,3 +46,8 @@
 	name = "Internal Energy Armblade Implant"
 	item_cost = 10
 	path = /obj/item/organ_module/active/simple/armblade/energy_blade
+
+/datum/uplink_item/item/implants/energy_blade
```

Comment:
Path is still `/datum/uplink_item/item/implants/energy_blade`

Reply:
Changed.